examples/cpp/Makefile

```diff
@@ -23,7 +23,7 @@ run: all
 all: build ${BIN}
 
 indent:
-	(find . ../../src/cpp ../../src/posix -name "*.cc" ; find ../../src -name "*.h" -maxdepth 0) | xargs clang-format-3.5 -i
+	(find . -name "*.cc") | xargs clang-format -i
```

Comment:
There is only dflags.h here (should be already formatted), so there is no need in .h